src/main/java/org/candlepin/subscriptions/db/model/Host.java

```diff
@@ -171,6 +172,7 @@ public void populateFieldsFromHbi(InventoryHostFacts inventoryHostFacts,
 
         if (inventoryHostFacts.getInventoryId() != null) {
             this.inventoryId = inventoryHostFacts.getInventoryId().toString();
+            this.instanceId = inventoryHostFacts.getInventoryId().toString();
```

Comment:
Is this supposed to be `getInventoryId()` again?

Reply:
Yeah, will add that as a comment in the code.